tests/dashboard/test_discounts.py

```diff
@@ -24,7 +24,8 @@ def test_vouchers_list(admin_client):
     assert response.status_code == 200
 
 
-def test_voucher_shipping_add(admin_client):
+def test_voucher_shipping_add(admin_client, settings):
+    settings.TIME_ZONE = "America/Chicago"
```

Comment:
Shouldn't it be set in `tests/settings.py` file for all tests globally? It seems like a workaround that we will have to copy-paste to other tests if they break for that reason?